### PR TITLE
Use CNI driver from cri-o/ocicni instead of internal one

### PR DESCRIPTION
**Pull request by dcbw**
@rajatchopra @mrunalp @runcom @Random-Liu

### CODE REVIEW

**Comment by mrunalp** [comment]
We have some conflicts.

**Comment by dcbw** [comment]
@mrunalp no longer :)

**Comment by mrunalp** [comment]
The code looks fine. We need to wait for jenkins to come back :)

File: server/sandbox_run.go [hunk 1]
```diff
@@ -464,27 +464,28 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
...
-			return nil, fmt.Errorf("failed to create network for container %s in sandbox %s: %v", containerName, id, err)
+		podNetwork := newPodNetwork(sb.Namespace(), sb.KubeName(), sb.ID(), netNsPath)
+		if err = s.netPlugin.SetUpPod(podNetwork); err != nil {
+			return nil, fmt.Errorf("failed to create network for container %s in sandbox %s: %v", sb.Name(), id, err)
 		}
 
-		if ip, err = s.netPlugin.GetContainerNetworkStatus(netNsPath, namespace, id, kubeName); err != nil {
-			return nil, fmt.Errorf("failed to get network status for container %s in sandbox %s: %v", containerName, id, err)
+		if ip, err = s.netPlugin.GetPodNetworkStatus(podNetwork); err != nil {
+			return nil, fmt.Errorf("failed to get network status for container %s in sandbox %s: %v", sb.Name(), id, err)
```
Comment (TomSweeneyRedHat) [inline server/sandbox_run.go]: Maybe I'm misunderstanding this change, but should the error message be tweaked?  Perhaps:

"failed to get network status for sandbox %s with id %v".  Or are sb.Name and containerName equivalent?

If a change is indeed needed here, similar comment for the other error messages below.

File: server/sandbox_run.go [hunk 2]
```diff
@@ -464,27 +464,28 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
...
 		if len(portMappings) != 0 {
 			ip4 := net.ParseIP(ip).To4()
 			if ip4 == nil {
-				return nil, fmt.Errorf("failed to get valid ipv4 address for container %s in sandbox %s", containerName, id)
+				return nil, fmt.Errorf("failed to get valid ipv4 address for container %s in sandbox %s", sb.Name(), id)
 			}
 
 			if err = s.hostportManager.Add(id, &hostport.PodPortMapping{
-				Name:         name,
+				Name:         kubeName,
```
Comment (rajatchopra) [inline server/sandbox_run.go]: sb.KubeName? Or sb.Name? 
Just to be consistent.

File: server/sandbox_run.go (hunk 1 above)
Comment (dcbw) [reply server/sandbox_run.go]: @TomSweeneyRedHat fixed

File: server/sandbox_run.go (hunk 2 above)
Comment (dcbw) [reply server/sandbox_run.go]: @rajatchopra you're right, fixed; and removed that naming consistency commit since it no longer really makes sense.

**Comment by rhatdan** [comment]
[retest]

**Comment by runcom** [comment]
> [retest]

CI is having some issues:
```
20:35:36 cat: temp.txt: No such file or directory
20:35:36 grep: PLATFORM/*/job.properties: No such file or directory
20:35:36 grep: PLATFORM/*/job.properties: No such file or directory
```

**Pull request merged**